drivers/misc/cldatadrv/rcar_cluster_drv.h

```diff
@@ -0,0 +1,26 @@
+#ifndef __RCAR_CLUSTER_DRV_H__
```

Comment:
Definitions in this file are not used by other files, so you safely can remove this `.h` file and move all the definitions to the `.c` file